### PR TITLE
Add episode 83

### PR DESCRIPTION
**Pull request by elyezer**

### CODE REVIEW

File: content/episodes/083-tarek-ziade.rst [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+de post é muito bem vindo.
+
+.. more
+
+Hello everyone and welcome to Castálio Podcast!
+
+Our guest for today is a Python developer currently working at Mozilla in the
+Services team, has written several books about Python, both in French and
+English, father of 3, is an avid runner, is a Vim user and works from Dijon,
+France. It is with a great pleasure that e welcome Tarek Ziadé to our podcast!
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
`that e welcome
+that we welcome
```

File: content/episodes/083-tarek-ziade.rst [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+First we started talking about Tarek's current role in the Mozilla's Services
+team. We could not start better since recently he have changed his activities
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-he have
+he has
```

File: content/episodes/083-tarek-ziade.rst [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+and started doing something different from the past few months he worked as a
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-something different from the past few months he worked
+something different spending these last few months in a new role as a
```

File: content/episodes/083-tarek-ziade.rst [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+manager. He told us that he is working on tool for the quality assurance of the
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-tool
+tools
```

File: content/episodes/083-tarek-ziade.rst [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+products Services team creates and maintain.
+
+Next we got a step back, and asked him how he has ended up working at Mozilla
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-got a step back
+ took a step back

-he has ended up
+he ended up
```

File: content/episodes/083-tarek-ziade.rst [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+and what he has done before working there. He told us that he was looking for a
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-he has done
+he had done
```

File: content/episodes/083-tarek-ziade.rst [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+company which does open source and the opportunity came when Mozilla was
+looking for a Python developer to work on porting some PHP projects to Python.
+Continuing on, he commented if there are many Python projects at Mozilla and
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-he commented if there are many Python projects at Mozilla and
- the good news is that
+he commented that
```

File: content/episodes/083-tarek-ziade.rst [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+the good news is that Python is very big in Mozilla. To finish this topic he
+told us about Rust at Mozilla (the Mozilla language).
+
+After that, we talked about what exactly he is doing to ensure the quality of
+the products. Tarek mentions that he worked on a project which helped
+orchestrate machines on Amazon Web Services and make hundreds of them to run a
+Python script in order to do a request to a web services and gather metrics.
+The other thing he's working on is to make sure all Mozilla's web services
+follows the `OpenAPI 2.0 specification`_ (aka Swagger), he highlight that the
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-follows the `OpenAPI 2.0 specification`_ (aka Swagger), he highlight that the
+follow the `OpenAPI 2.0 specification`_ (aka Swagger). He highlighted that the

File: content/episodes/083-tarek-ziade.rst [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+QA tools can get information and do some automatic work if the API follows that
+specification, to complement, Bruno mentioned about his project `Flasgger`_
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-specification, to complement,
+specification. To complement,
```

File: content/episodes/083-tarek-ziade.rst [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+and control processes and sockets. The project was created in order avoid
+struggling with `Supervisor`_. Now the project is not being used since they
+moved to deploy the project on the cloud and each application runs on its own
+virtual machine, so there's no need to control many process on the same
+machine. It does not mean that the project is dead, it still active and
+maintained but not so actively as it used to be.
+
+As we mentioned in the beginning, Tarek has written a good book about Python
+`Expert Python Programming`_ (he mentions that he was not involved on the
+second edition because one of his child was born). And we can expect the same
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-his child
+his children
```

File: content/episodes/083-tarek-ziade.rst [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+or better quality for his next book, `Python Microservices Development`_ which
+is available to pre-order and is expected to be published in September 2017.
+Tarek gave us an overview of what he is going to cover on the new book.
+
+Moving away from the technical conversation for a while, Tarek tell us about
+his passion to running and participating of a marathon. Also he gives some tips
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-passion to running
+passion for running
```

File: content/episodes/083-tarek-ziade.rst [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+about how he prepares and some extra activities that helps on getting not only
+the body but also the brain ready for running long distances.
+
+Our last question was about Tarek's opinion about Python 3. He comments that
+sometimes we need to pick a version of Python based on the Operating System
+because not all them have Python 3, for example, Cent OS. He mentions that two
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-Cent OS
+CentOS
```

File: content/episodes/083-tarek-ziade.rst [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+or three years ago we didn't have many libraries supporting Python 3 and now we
+do. Also, he thinks Python 3 does not offers a really killer feature but
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-does not offers a really killer feature but
-does not offer a killer feature other than perhaps
```

File: content/episodes/083-tarek-ziade.rst [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+asyncio to push the wish of people to upgrade, which could be slowing down the
```
Comment (omaciel) [inline content/episodes/083-tarek-ziade.rst]:
```diff
-to push the wish of people
+to convince people
```

**Comment by elyezer** [comment]
Handled all comments

**Comment by rochacbruno** [comment]
The missing musician name is:

Chali 2na - http://www.last.fm/pt/music/Chali+2na

(still misses one which is the canandian eletronic musician, I am trying to discover)

**Comment by omaciel** [comment]
@rochacbruno all yours :)

**Pull request merged**